src/github-repo.ts

```diff
@@ -244,7 +245,17 @@ export class GithubRepo {
     }
     // Doing in sequence to not overload GitHub with requests
     for (const asset of assets) {
-      await this._uploadAsset(releaseDetails, asset);
+      try {
+        await this._uploadAsset(releaseDetails, asset);
+      } catch (err) {
+        if ((err as OctokitRequestError)?.status === 500) {
```

Comment:
Optional, but I think it makes sense to do this for all 5xx errors since they more or less have the same semantics on the client side:

```suggestion
        const status = (err as OctokitRequestError)?.status;
        if (status >= 500 && status <= 599) {
```